README.md

```diff
@@ -35,7 +35,7 @@ We'd like you to output 3 things:
 
 2) Spend around 2 hours on this, don't feel you have to rush a solution. **You will not be judged on whether you finish or not**. We would much rather you **give us a solution that you are proud of** and feel best displays the type of code you would want to ship into production. That said, make sure there is a way to run your app and it does give some sort of output.
 
-3) Tests are not a hard requirement but are strongly encouraged, if you're not used to testing your code have a go at it anyway. It's not that hard to get started with it and we promise it will be a great skill for you to learn.
+3) Tests are not a hard requirement but are strongly encouraged.

```

Comment:
We quite liked this line when going through it before - how come you think it's best to remove it?